playbooks/roles/tor-bridge/tasks/main.yml

```diff
@@ -27,8 +27,7 @@
 - import_tasks: firewall.yml
 
 - name: Generate a random Nickname for the bridge
-  shell: grep -v -P "[\x80-\xFF]" /usr/share/dict/american-english-huge | sed -e "s/'//" | shuf -n 2 | xargs | sed -e 's/ //' | cut -c 1-16 > {{ tor_bridge_nickname_file }}
-  args:
+  shell: "{{ streisand_word_gen.tor_nickname }} > {{ tor_bridge_nickname_file }}  args:
```

Comment:
You lost a `"` at the end of the `shell:` item and pulled up the `args:` item here. It's breaking CI: 
```
ERROR! Syntax Error while loading YAML.
The error appears to have been in '/home/travis/build/StreisandEffect/streisand/playbooks/roles/tor-bridge/tasks/main.yml': line 31, column 15, but may
be elsewhere in the file depending on the exact syntax problem.
The offending line appears to be:
  shell: "{{ streisand_word_gen.tor_nickname }} > {{ tor_bridge_nickname_file }}  args:
    creates: "{{ tor_bridge_nickname_file }}"
              ^ here
```

Reply:
Thanks. This is motivating me to turn on all the YAML validation stuff in my editor. Right now the validator is unhappy with long lines, but I think I'll turn that off.